frontend/public/components/utils/headings.tsx

```diff
@@ -176,7 +176,7 @@ export const ConnectedPageHeading = connectToModel(
                   )}
                 </span>
                 {resourceStatus && (
-                  <ResourceStatus additionalClassNames="hidden-xs">
+                  <ResourceStatus additionalClassNames="pf-v6-u-display-none pf-v6-u-display-block-on-sm">
```

Comment:
this seems working fine too
<img width="920" height="797" alt="Screenshot 2025-07-11 at 12 12 31 PM" src="https://github.com/user-attachments/assets/b0bb6aa7-dd8f-4fd5-8589-124a0bb4b133" />